src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -120,13 +120,13 @@ public FromBase64Transform(FromBase64TransformMode whitespaces)
             _whitespaces = whitespaces;
         }
 
-        // Converting from Base64 generates 3 bytes output from each 4 bytes input block
-        private const int Base64InputBlockSize = 4;
         // A buffer with size 32 is stack allocated, to cover common cases and benefit from JIT's optimizations.
         private const int StackAllocSize = 32;
-        public int InputBlockSize => 1;
+
+        // Converting from Base64 generates 3 bytes output from each 4 bytes input block
+        public int InputBlockSize => 4;
         public int OutputBlockSize => 3;
-        public bool CanTransformMultipleBlocks => false;
+        public bool CanTransformMultipleBlocks => true;
```

Comment:
> though that means it'll almost never use the stackalloc buffers
(from https://github.com/dotnet/corefx/issues/39583#issuecomment-512854180)

Are there any statistics about the typical `inputCount` (i.e. stream length on which CryptoStream should operate)? With these numbers the buffers could be tweaked (stackalloc / renting).

If not, I believe the current implementation of stackalloc / renting is still good.

Reply:
> Are there any statistics about the typical inputCount (i.e. stream length on which CryptoStream should operate)? 

Nope.

The numbers I do have: apisof.net reports 1% APIPort usage for FromBase64Transform, and 0.4% for ToBase64Transform (25% CryptoStream)... and RC2 has almost more usage than that (somehow).  Convert.ToBase64String(byte[]) has 51.4% usage, and Convert.FromBase64String is 43,8% usage.

So "no idea what the inputs into this are, but it's definitely not the common way of doing Base64".